vtiger_connector_sales/models/res_company.py

```diff
@@ -157,33 +169,40 @@ def sync_vtiger_sale_Quotes(self):
             if result.get('success'):
                 self.update_existing_sale_Quotes(result)
```

Comment:
check update_existing_sale_order and update_existing_sale_Quotes methods, they both are having exact same code